builtin/mainmenu/tab_multiplayer.lua

```diff
@@ -20,7 +20,13 @@ local function get_formspec(tabview, name, tabdata)
 	-- Update the cached supported proto info,
 	-- it may have changed after a change by the settings menu.
 	common_update_cached_supp_proto()
-	local fav_selected = menudata.favorites[tabdata.fav_selected]
+	local fav_selected = {}
```

Comment:
`fav_selected` is always overwritten with something. No need to initialize the table.